cmd/webhook/main.go

```diff
@@ -14,7 +14,41 @@
 
 package main
 
-func main() {
+import (
+	"flag"
+
+	"antrea.io/resource-auditing/pkg/webhook"
+	"antrea.io/resource-auditing/pkg/webhook/gitmanager"
+	"k8s.io/klog/v2"
+)
+
+func processArgs(portFlag *string, dirFlag *string) {
+	flag.StringVar(portFlag, "p", "8080", "specifies port that audit webhook listens on")
+	flag.StringVar(dirFlag, "d", "", "specifies directory where resource repository is created")
+	flag.Parse()
+}
```

Comment:
`portFlag` and `dirFlag` are global variables, you don't need to pass them as arguments